Hello the automatic script doesn't work anymore, I have this error :

Hello the automatic script doesn't work anymore, I have this error :

`
yarn start *** *** 0 ***
  shell: /bin/bash -e {0}
yarn run v1.22.4
$ node claimer.js *** *** 0 ***
 2020-08-16 | 18:59:22.724 | ERROR | { Error: ENOENT: no such file or directory, mkdir '/home/runner/work/claim-epic-freebies/claim-epic-freebies/.egstore/b60e62be8f01d1f927f45690555ccd86ca6edcd8a0f8c97b0466048ba97acc02'
    at Object.mkdirSync (fs.js:757:3)
    at new Launcher (/home/runner/work/claim-epic-freebies/claim-epic-freebies/node_modules/epicgames-client/src/Client/index.js:108:10)
    at /home/runner/work/claim-epic-freebies/claim-epic-freebies/claimer.js:70:26
    at process._tickCallback (internal/process/next_tick.js:68:7)
  errno: -2,
  syscall: 'mkdir',
  code: 'ENOENT',
  path:
   '/home/runner/work/claim-epic-freebies/claim-epic-freebies/.egstore/b60e62be8f01d1f927f45690555ccd86ca6edcd8a0f8c97b0466048ba97acc02' }
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
##[error]Process completed with exit code 1.
`

--- a/.github/workflows/claim.yml
+++ b/.github/workflows/claim.yml
@@ -29,5 +29,8 @@ jobs:
             - name: Install dependencies
               run: yarn
 
+            - name: Hotfix for \#4 # https://github.com/xuac/claim-epic-freebies/issues/4
+              run: mkdir .egstore
+
             - name: Get Epic Freebies
               run: yarn start ${{ secrets.USERNAME }} ${{ secrets.PASSWORD }} 0 ${{ secrets.SECRETKEY }}

